noir-projects/aztec-nr/aztec/src/oracle/notes.nr

```diff
@@ -198,3 +198,15 @@ pub unconstrained fn check_nullifier_exists(inner_nullifier: Field) -> bool {
 
 #[oracle(checkNullifierExists)]
 unconstrained fn check_nullifier_exists_oracle(_inner_nullifier: Field) -> Field {}
+
+/// Returns the tagging secret for a given sender and recipient pair. For this to work, PXE must know the ivpsk_m of the sender.
+/// For the recipient's side, only the address is needed.
+pub unconstrained fn get_tagging_secret(sender: AztecAddress, recipient: AztecAddress) -> Field {
+    get_tagging_secret_oracle(sender, recipient)
+}
```

Comment:
Let's remove `sender` from this function: you should never be able to call this with any sender that is not the address currently executing code. Otherwise we'd be revealing secret information about a seperate contract, defeating the entire point of siloing.

Reply:
This is wrong: `sender` can be any account, since this oracle call will fail anyway if PXE doesn't know `sender`'s secret (`ivsk`). If multiple accounts reside on the same device, then that's a PXE siloing concern.

The implicit argument here is `app`, not `sender`.